visual-java/src/main/java/com/saucelabs/visual/VisualApi.java

```diff
@@ -190,6 +201,15 @@ public void setCaptureDom(Boolean captureDom) {
     this.captureDom = captureDom;
   }
 
+  /**
+   * Enables full page screenshots
+   *
+   * @param fullPageScreenshotConfig activation of DOM Capture.
+   */
+  public void setFullPageScreenshotConfig(FullPageScreenshotConfig fullPageScreenshotConfig) {
```

Comment:
I see that this is aligned with the way captureDom is handled but in CheckOptions there are two functions provided for enabling FPS called `enableFullPageScreenshots()` and `enableFullPageScreenshots(FullPageScreenshotConfig config)`. This arguably reads nicer (as it's a user facing API) and provides a way to enable basic FPS without passing a config. I suggest we do the same here

Reply:
Changed. Good idea, thanks.